frontend/src/components/ContextMenu.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import {
+  EuiButton,
+  EuiContextMenuPanel,
+  EuiContextMenuItem,
+  EuiPopover,
+  EuiCopy,
+  useGeneratedHtmlId, EuiButtonEmpty,
+} from '@elastic/eui';
+import {Recording, useDeleteRecordingById} from "../api";
+import {useQueryClient} from "@tanstack/react-query";
+
+export default ({recordingId, deleteCallback}: {recordingId: string, deleteCallback: any}) => {
```

Comment:
Additionally, make sure to properly specify the type of `deleteCallback`. Relevant documentation: https://www.typescriptlang.org/docs/handbook/2/functions.html